bsp008z-pressure-sensor/include/PressureSensor.h

```diff
@@ -0,0 +1,17 @@
+#ifndef BSP008Z_PRESSURE_SENSOR_H
+#define BSP008Z_PRESSURE_SENSOR_H
+
+#include <AnalogPinReader.h>
+
+namespace wlp {
+    namespace BSP008Z {
+        class PressureSensor: public AnalogPinReader {
+        public:
+            explicit PressureSensor(Board::AnalogPin analog_pin)
+            : AnalogPinReader(analog_pin) {};
+            double read_value();
```

Comment:
same deal with #2, mark this as `override` once the parent function has been changed to `virtual`